src/main/java/eu/europa/ted/efx/sdk1/EfxExpressionTranslator08.java

```diff
@@ -42,7 +42,10 @@
 import eu.europa.ted.efx.model.Expression.StringListExpression;
 import eu.europa.ted.efx.model.Expression.TimeExpression;
 import eu.europa.ted.efx.model.Expression.TimeListExpression;
-import eu.europa.ted.efx.sdk0.v8.EfxParser.*;
+import eu.europa.ted.efx.sdk1.EfxBaseListener;
+import eu.europa.ted.efx.sdk1.EfxLexer;
+import eu.europa.ted.efx.sdk1.EfxParser;
```

Comment:
Those 3 import statements are not needed: they refer to classes in the same package as the current class.

Reply:
Fixed